gradle-palantir-java-format/src/main/java/com/palantir/javaformat/gradle/JavaFormatExtension.java

```diff
@@ -16,22 +16,30 @@
 
 package com.palantir.javaformat.gradle;
 
+import com.google.common.base.Suppliers;
 import com.google.common.collect.Iterables;
 import com.palantir.javaformat.java.FormatterService;
 import java.net.MalformedURLException;
 import java.net.URL;
 import java.net.URLClassLoader;
 import java.util.ServiceLoader;
+import java.util.function.Supplier;
 import org.gradle.api.artifacts.Configuration;
 
 public class JavaFormatExtension {
```

Comment:
:+1: confirmed this is only registered by the PalantirJavaFormatProvider plugin, which only works on the root project:

```
        Preconditions.checkState(
                rootProject == rootProject.getRootProject(),
                "May only apply com.palantir.java-format-provider to the root project");
```